.github/workflows/gh_action_lint.yaml

```diff
@@ -0,0 +1,17 @@
+name: YAML Lint
+
+on: # yamllint disable-line rule:truthy
+  push
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: YAML Lint github actions
+        uses: ibiqlik/action-yamllint@v3
+        with:
+          file_or_dir: .github
```

Comment:
Also we'll want to run yamllint across not just the .github folder but every yaml file in the repo so I think we want `file_or_dir: .` here.

Reply:
My thinking is: the .github folder might require different settings that the rest of the code, therefore it might make sense to have one step specifically for that, and then another step for the rest of the code.

Reply:
OK this is probably over-engineered :D will move it back to linting the whole repo :)